### PR TITLE
Create skeleton user guide

### PR DESCRIPTION
**Pull request by teo-jun-xiong**

### CODE REVIEW

**Review by hans555 — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -14,31 +14,28 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/AY1920S2-CS2103T-F11-4/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team F11-4`      Since: `Jun 2016`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+Cooking Papa is for those who prefer to use a desktop app for managing recipes. More importantly, Cooking Papa is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Cooking Papa can help you plan and cook meals more easily than traditional GUI apps. Interested? Jump to the Section 2, “Quick Start” to get started. Enjoy!
```
Comment (gohkh) [inline docs/UserGuide.adoc]: I think removing the angle brackets breaks the link to section 2.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -52,110 +49,45 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
-=== Viewing help : `help`
+=== Ingredients inventory commands
```
Comment (gohkh) [inline docs/UserGuide.adoc:52]: Maybe we can just remove all the commands for now, because we are changing some commands to make them easier to use

Comment (teo-jun-xiong) [reply docs/UserGuide.adoc:52]: ok I left them in as a guideline, but I think we can simply check the sample's. reverted.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (teo-jun-xiong) [reply docs/UserGuide.adoc]: Added the angle brackets back in

**Review by gohkh — APPROVED** [review]

**Pull request merged**